docs/servers/find-server-ip-address.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Find your Server IP Address
+intro: Having the server ip address enables you and your team to interact with your server via Secure Shell Protocol (SSH) access or configure a Domain Name Server (DNS) record, so your users can access your applications using a domain name instead.
```

Comment:
1. Although I support the idea of explaining acronyms before using them in text, I believe "DNS record", SSH and SSL are well spread acronyms across the IT community and, specially, giving the context of server and SSL configuration perhaps the user can get more help searching for the acronyms themselves than using the long version of each of them.
2. Not all web application are meant to be accessed by human beings, so I suggest remove the text `so your users can access your applications`, as this might not be accurate and some users may not relate to it
3. Not sure if we can use bullet lists in the intro text. If not, the list can be replaced with and enumeration of ideas prefixed with other terms such as `e.g.` and `such as` itself 

What do you think?

```suggestion
intro: Having the IP address of a server is useful when performing operations that require network connections to the server.
Example of server network operations include:
- log to your server via SSH to run commands in a terminal
- configure a DNS record so your application can accessed using a domain name
```

Reply:
> Although I support the idea of explaining acronyms before using them in text, I believe "DNS record", SSH and SSL are well spread acronyms across the IT community and, specially, giving the context of server and SSL configuration perhaps the user can get more help searching for the acronyms themselves than using the long version of each of them.

Agreed

> Not all web application are meant to be accessed by human beings, so I suggest remove the text so your users can access your applications, as this might not be accurate and some users may not relate to it

Agreed

> Not sure if we can use bullet lists in the intro text. If not, the list can be replaced with and enumeration of ideas prefixed with other terms such as e.g. and such as itself

There's a workaround: from what I've seen the intro frontmatter does not support the full Markdown syntax, so I'll move the other paragraphs to the content start

Reply:
Currently, the unordered list doesn't have specific styling rules, so I'll use the other suggested approach

![image](https://github.com/devopness/devopness/assets/33532177/df425342-7d17-4191-bd6f-ce8a18e5dbcb)

Reply:
Changes made

![image](https://github.com/devopness/devopness/assets/33532177/43626691-3da2-4030-aac5-64522270fc86)